### ISSUE TITLE
[Bug?] VLC opens multiple windows instead of queueing the videos in the playlist

### ISSUE THREAD

**Issue by ushuc** [gpodder/gpodder#1478]
When I select multiple episodes in gPodder and choose "Add to playlist -> VLC" in the context menu, then VLC open multiple windows (one for each selected episode) instead of opening one window with all the selected episodes queued in the playlist.

What am I doing wrong? Or is it a bug?

**Comment by auouymous** [comment]
Which version of gpodder do you have and what OS? How do I get the `Add to playlist` in context menu? I am unable to find that string in gpodder 3.11.1.

**Comment by auouymous** [comment]
Did this work in previous versions of gpodder and is now an issue in 3.11.1?

I don't have VLC to test this but it works fine with mplayer and mpv in Linux. It is possible this is an issue with the `HKEY_CLASSES_ROOT\VLC.mp3\shell\AddToPlaylistVLC\command` command the extension uses to launch VLC on Windows.

**Comment by auouymous** [comment]
> Do I maybe have to add some variables to the command line here?

No, the enqueue extension has its own list of commands. Is there an option in VLC called "allow only one instance", and do you have that unchecked? The VLC command in the enqueue extension might have been added by someone with that setting checked, so it worked for them. There is probably a flag the extension could pass to temporarily enable this setting, but someone with Windows would need to figure it out.

**Comment by auouymous** [comment]
We just need to figure out why the `AddToPlaylistVLC` in the VLC command is not working.

It wouldn't be too difficult to pass an URL if not downloaded. The code that determines if an episode can be streamed could be used to pass URLs.
